scripts/pylib/pytest-twister-harness/src/twister_harness/fixtures.py

```diff
@@ -58,6 +60,9 @@ def dut(request: pytest.FixtureRequest, device_object: DeviceAdapter) -> Generat
 def shell(dut: DeviceAdapter) -> Shell:
     """Return ready to use shell interface"""
     shell = Shell(dut, timeout=20.0)
+    if prompt := find_in_config(Path(dut.device_config.app_build_dir) / 'zephyr' / '.config',
```

Comment:
how does this work with a sysbuild project?

Reply:
`app_build_dir` path is updated with the default domain name read from domains.yaml file. Find implementation details here: https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/pytest-twister-harness/src/twister_harness/twister_harness_config.py#L39